components/SplitPane/library/src/commonMain/kotlin/org/jetbrains/compose/splitpane/SplitPaneDSL.kt

```diff
@@ -83,12 +86,18 @@ interface SplitterScope {
 internal class HandleScopeImpl(
     private val containerScope: SplitPaneScopeImpl
 ) : HandleScope {
-    override fun Modifier.markAsHandle(): Modifier = this.pointerInput(containerScope.splitPaneState) {
-        detectDragGestures { change, _ ->
-            change.consume()
-            containerScope.splitPaneState.dispatchRawMovement(
-                if (containerScope.isHorizontal) change.position.x else change.position.y
-            )
+    @Composable
+    override fun Modifier.markAsHandle(): Modifier = this.run {
```

Comment:
```suggestion
    override fun Modifier.markAsHandle(): Modifier = composed {
```
You can access `LocalLayoutDirection` this way.

Better though to convert it to [Modifier.Node](https://developer.android.com/jetpack/compose/custom-modifiers), but it is not easy - `composed` will do for now.

Reply:
done, also modified it in the Implementaion